### PR TITLE
docs: rm type hints in ignite.contrib.handlers (2)

### PR DESCRIPTION
**Pull request by ydcjeff**
Part of #1665 

Description:

Check list:

- [ ] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [x] Documentation is updated (if required)

### CODE REVIEW

File: ignite/contrib/handlers/param_scheduler.py [hunk 1]
```diff
@@ -194,7 +194,7 @@ def plot_values(cls, num_events: int, **scheduler_kwargs: Mapping) -> Any:
 
         Args:
             num_events: number of events during the simulation.
-            **scheduler_kwargs : parameter scheduler configuration kwargs.
+            scheduler_kwargs : parameter scheduler configuration kwargs.
```
Comment (vfdev-5) [inline ignite/contrib/handlers/param_scheduler.py:197]:
```suggestion
            scheduler_kwargs: parameter scheduler configuration kwargs.
```

File: ignite/contrib/handlers/param_scheduler.py [hunk 2]
```diff
@@ -589,6 +589,7 @@ def simulate_values(  # type: ignore[override]
             durations: list of number of events that lasts a parameter scheduler from schedulers.
             param_names: parameter name or list of parameter names to simulate values.
                 By default, the first scheduler's parameter name is taken.
+            kwargs:
```
Comment (vfdev-5) [inline ignite/contrib/handlers/param_scheduler.py:592]: this is a good question to what corresponds those kwargs

File: ignite/contrib/handlers/param_scheduler.py [hunk 3]
```diff
@@ -774,7 +775,7 @@ def create_lr_scheduler_with_warmup(
             by 100 simulated learning rate values.
 
     Returns:
-        ConcatScheduler: learning rate scheduler with linear warm-up.
+        learning rate scheduler with linear warm-up.
```
Comment (vfdev-5) [inline ignite/contrib/handlers/param_scheduler.py:778]: Maybe, it's better to keep ConcatScheduler only ?

File: ignite/contrib/handlers/param_scheduler.py [hunk 4]
```diff
@@ -886,7 +887,7 @@ class PiecewiseLinear(ParamScheduler):
         param_group_index: optimizer's parameters group to use.
 
     Returns:
-        PiecewiseLinear: piecewise linear scheduler
+        piecewise linear scheduler instance
```
Comment (vfdev-5) [inline ignite/contrib/handlers/param_scheduler.py:890]: Same here ?

File: ignite/contrib/handlers/param_scheduler.py [hunk 5]
```diff
@@ -1083,11 +1085,12 @@ def simulate_values(cls, num_events: int, schedulers: List[_LRScheduler], **kwar
         """Method to simulate scheduled values during num_events events.
 
         Args:
-            num_events (int): number of events during the simulation.
-            schedulers (subclass of `torch.optim.lr_scheduler._LRScheduler`): lr_scheduler object to wrap.
+            num_events: number of events during the simulation.
+            schedulers: lr_scheduler object to wrap.
+            kwargs:
```
Comment (vfdev-5) [inline ignite/contrib/handlers/param_scheduler.py:1090]:
```suggestion
            kwargs: kwargs passed to construct an instance of :class:`ignite.contrib.handlers.ParamGroupScheduler`.
```

**Review by vfdev-5 — COMMENTED** [review]
Thanks a lot for working on that issue @ydcjeff !
I left few comments

File: ignite/contrib/handlers/param_scheduler.py (hunk 2 above)
Comment (ydcjeff) [reply ignite/contrib/handlers/param_scheduler.py:592]: Yes, `kwargs` is used in
```py
scheduler = cls(schedulers=schedulers, save_history=False, durations=durations, **kwargs)
```
`cls` is `ConcatScheduler` which only has 3 arguments, so kwargs is unnecessary?

Comment (vfdev-5) [reply ignite/contrib/handlers/param_scheduler.py:592]: Yes, seems like that. Let's remove it from docstring and the function signature

**Review by vfdev-5 — COMMENTED** [review]
Looks good ! A nit to remove kwargs from concat scheduler and good to go, I think.

**Review by vfdev-5 — APPROVED** [review]
